Tests/Common/Util/ExtensionsTests.cs

```diff
@@ -297,6 +300,73 @@ public void RoundsDownInTimeZone()
             Assert.AreEqual(time, roundedTime);
         }
 
+        [Test, Ignore]
+        public void TryConvertPyObject()
```

Comment:
This test appears to be many many tests in one. A common saying is `one assert per unit test` -- but I like to generalize this to `one *logical* assert per unit test` -- both of which are somewhat fancy ways of saying a test should only test one behavior. Maybe these cases can be extracted into `[TestCase]`s ?